src/include/sof/audio/chain_dma.h

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2022 Intel Corporation. All rights reserved.
+//
+// Author: Piotr Makaruk <piotr.makaruk@intel.com>
+
```

Comment:
Consider using 'include guard' for this file.

`#ifndef __SOF_CHAIN_DMA_H__`
`#define __SOF_CHAIN_DMA_H__`
`#endif /* __SOF_CHAIN_DMA_H__ */`

Reply:
I think @tlissows means this:
```
#ifndef __SOF_CHAIN_DMA_H__
#define __SOF_CHAIN_DMA_H__

  < entire file goes here >

#endif /* __SOF_CHAIN_DMA_H__ */
```